se/se_epub_build.py

```diff
@@ -11,7 +11,6 @@
 import shutil
 import subprocess
 import tempfile
-from distutils.dir_util import copy_tree # pylint: disable=deprecated-module
```

Comment:
`distutils.copy_tree` is used in other places in this code. If you're removing the import then you have to replace the other instances with `shutil.copytree`. You must also add the ignore errors flag in all cases too because that's the behavior we're expecting. 

It would be helpful to test these changes before submitting a PR!

Reply:
Sorry, I missed the other usage of copy_tree. I fixed it. I also added the ignore flag to all the invocations that I replaced.

Ran the tests locally and they passed (see output below). Let me know if there is a specific test that you would like to run.

```
$ $HOME/.local/pipx/venvs/standardebooks/bin/pytest
============================================================================================================ test session starts =============================================================================================================
platform darwin -- Python 3.12.0, pytest-7.3.1, pluggy-1.3.0
rootdir: /Users/deepak/code/tools
collected 43 items

tests/test_build.py .                                                                                                                                                                                                                  [  2%]
tests/test_build_toc.py ....                                                                                                                                                                                                           [ 11%]
tests/test_formatting.py ...............                                                                                                                                                                                               [ 46%]
tests/test_lint.py ......                                                                                                                                                                                                              [ 60%]
tests/test_simple_cmds.py ..........                                                                                                                                                                                                   [ 83%]
tests/test_text_cmds.py .......                                                                                                                                                                                                        [100%]

============================================================================================================ 43 passed in 20.58s =============================================================================================================
```